lib/index.js

```diff
@@ -250,6 +254,10 @@ function initOptions(options) {
     ...options.ownerChanged ? ['ownerChanged'] : []
   ]
 
+  if (options.deep) {
+    options.packageFile = deepPattern
+  }
```

Comment:
We want to avoid unnecessary mutations; let's set `packageFile` in the merged return object below.